apache2/waf_logging/waf_log_util.cc

```diff
@@ -177,41 +178,42 @@ static std::string get_json_log_message(const char* resource_id, const char* ope
     const str_view waf_unique_id_str = make_string_view_strip_quotes(waf_unique_id);
 
     return to_json_string({
-            resource_id,
-            operation_name,
-            category,
-            waf_logging::property {
-                instance_id,
-                client_ip,
-                client_port,
-                request_uri,
-                rule_set_type,
-                rule_set_version,
-                rule_id_str,
-                message_str,
-                action_str,
-                site_str,
-                waf_logging::detail {
-                    details_messages,
-                    details_data_str,
-                    details_file_str,
-                    details_line_str
-                },
+        timestamp,
```

Comment:
Note: the change here is the indenting. It was off.
If we don't want these kinds of formatting fixes then I'm okay with reverting this

Reply:
Minor formatting changes are fine, I just thought that the change was in an extra comma that looked redundant.

Reply:
On a broader note, I generally prefer that non-functional changes (style, formatting, comments) that are not directly related to the current fix be published separately, but when the changes are minor (few lines only) it is apparently easier to sneak it in along with the major changes.

Reply:
sounds good